policy/release/tasks.rego

```diff
@@ -197,11 +196,40 @@ current_required_tasks := task_data if {
 
 ## get the required task data for a pipeline with a label
 required_pipeline_task_data := task_data if {
-	some att in lib.pipelinerun_attestations
+	some att in _pipelinerun
 	count(tkn.tasks(att)) > 0
 	task_data := tkn.required_task_list(att)
 }
 
-_status(task) := status if {
+_status(task) = status if {
+	not task.status.conditions
 	status := task.status
+} else = status if {
+	some condition in task.status.conditions
+	condition.type == "Succeeded"
+	status := _slsav1_status(condition)
 } else = "MISSING"
+
+_slsav1_status(condition) := status if {
+	condition.status == "True"
+	status := "Succeeded"
+}
+
+_slsav1_status(condition) := status if {
+	condition.status == "False"
+	status := "Failed"
+}
+
+_pipelinerun := att if {
```

Comment:
I think I can move this or something similar to the `attestations.rego` lib, but wanna do that in a separate PR.